tests/containers/docker_image.pm

```diff
@@ -38,7 +38,7 @@ sub run {
 
     # We may test either one specific image VERSION or comma-separated CONTAINER_IMAGES
     my $versions   = get_var('CONTAINER_IMAGE_VERSIONS', get_required_var('VERSION'));
-    my $dockerfile = ($host_distri ne 'suse') ? 'Dockerfile.python3' : 'Dockerfile';
+    my $dockerfile = !check_os_release('suse', 'PRETTY_NAME') ? 'Dockerfile.python3' : 'Dockerfile';
```

Comment:
I'm not sure, but for me it took me a time to understand it. What about be more explicit, so it is easy to read like:
```suggestion
    my $dockerfile = $host_distri !~ m/^(sle|opensuse-.*)$/ ? 'Dockerfile.python3' : 'Dockerfile';
```

Reply:
`check_os_release` has been tested extensively IMO and is quite solid. I would rather leave it as it is. However your suggestion might be faster. but i think it doesnt matter here.

Reply:
I agree with Clemens. We already have `$host_distri` so let's use it please.

Reply:
thats funny argument as we already use `check_os_release` too. which means that we do not have to import anything mew too. tespa. i did the change. *this change is ready for review*